app/views/coordinate.scala

```diff
@@ -75,8 +75,10 @@ object coordinate {
         ),
         div(cls := "coord-trainer__board main-board")(
           svgTag(cls := "coords-svg", viewBoxAttr := "0 0 100 100")(
-            textTag(cls := "coord current-coord"),
-            textTag(cls := "coord next-coord")
+            textTag(cls := "coord coord_resolved"),
+            textTag(cls := "coord coord_current"),
+            textTag(cls := "coord coord_next"),
+            textTag(cls := "coord coord_new")
```

Comment:
It's better to stick to the existing naming convention or at least use proper BEM, i.e. `coord--current`, `coord--next`, etc.

Reply:
> It's better to stick to the existing naming convention

I tried to find the naming convention for the project, but no luck. Can you tell me where to find it?

> or at least use proper BEM, i.e. coord--current, coord--next, etc.

Actually, that's exactly what I did - according to the [BEM Naming rules](https://en.bem.info/methodology/naming-convention/#naming-rules), `"The modifier name is separated from the block or element name by a single underscore (_)."`. And the a double hyphen variant comes from [one of the alternative naming schemes](https://en.bem.info/methodology/naming-convention/#two-dashes-style).

Anyways, I'm changing the class names if it's according to the project's naming convention.

Reply:
Interesting, I didn't know that. But honestly, I've never seen that BEM style.

Anyway, I don't think we have a fixed naming convention for CCS. There are even some uses of underscores here and there though not for modifiers but it's better to avoid that in new code imo. Most of the time, we just leave out the base class or use something more natural, like the original `next-coord`.